src/Response/SearchCity.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MeestBundle\Response;
+
+use Answear\MeestBundle\Response\DTO\CityDTO;
+
+class SearchCity implements ResponseInterface
+{
+    /**
+     * @var CityDTO|CityDTO[]
```

Comment:
This looks off, can the API really return single object or an array?

Reply:
Yes, for Lviv it returned 8 elements iirc, so I decided to keep it this way.

Reply:
But the library's API should be clean i.e. it should always be an array (with 0, 1, or more elements) to prevent adding `is_array` calls all over the place